tools/rapids-configure-rattler

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# A utility script that sets variables needed for `rattler-build`
+
+# We only want the last two (of three) entries from splitting the `git describe` output on `-`
+# so we purposefully overwrite `GIT_DESCRIBE_NUMBER` to avoid setting an unwanted variable
```

Comment:
Thanks for this comment, because the duplication is confusing! And I do get that anything else we might do has a non-0 chance of conflicting with something else in the environment, e.g. like this:

```shell
IFS=- read -r __ignore_me GIT_DESCRIBE_NUMBER GIT_DESCRIBE_HASH <<< "$(git describe --tags)"
unset __ignore_me
```

I'm not familiar enough with `bash` to know if there's something else that could be done to here, so I think this approach + the comment is a good one.